docs/spec/spec.md

```diff
@@ -3,11 +3,17 @@
 _Owners_: @daneshk @kalaiyarasiganeshalingam                                      
 _Reviewers_: @daneshk  
 _Created_: 2021/12/12  
-_Updated_: 2021/12/12   
+_Updated_: 2022/02/10   
```

Comment:
We need to include the edition as well.

https://github.com/ballerina-platform/module-ballerina-http/blob/master/docs/spec/spec.md